src/app.ts

```diff
@@ -881,6 +891,16 @@ app.post(
     }),
 );
 
+app.post(
```

Comment:
This is manually duplicated because webhooks aren't migrated to the `@Route` decorator yet, as they have special middleware